tests/unit/test_main.py

```diff
@@ -1,22 +1,36 @@
+import os
+
 from click.testing import CliRunner
 
-from scanapi.main import main
+from scanapi.main import run, convert
 
 
 class TestMain:
-    def test_call_save_preferences(self, mocker):
-        runner = CliRunner()
-        mock_save_preferences = mocker.patch(
-            "scanapi.settings.Settings.save_preferences"
-        )
-        result = runner.invoke(main, ["--output-path", "my_output.html"])
-
-        assert result.exit_code == 4
-        mock_save_preferences.assert_called_once_with(
-            **{
-                "spec_path": None,
-                "output_path": "my_output.html",
-                "config_path": None,
-                "template": None,
-            }
-        )
+    class TestRun:
+        def test_call_save_preferences(self, mocker):
+            runner = CliRunner()
+            mock_save_preferences = mocker.patch(
+                "scanapi.settings.Settings.save_preferences"
+            )
+            result = runner.invoke(run, ["--output-path", "my_output.html"])
+
+            assert result.exit_code == 4
+            mock_save_preferences.assert_called_once_with(
+                **{
+                    "spec_path": None,
+                    "output_path": "my_output.html",
+                    "config_path": None,
+                    "template": None,
+                }
+            )
+
+    class TestConvert:
+        def test_should_call_openapi_to_yaml(self, mocker):
+            runner = CliRunner()
+            mock_openapi_to_yaml = mocker.patch("scanapi.main.openapi_to_yaml")
```

Comment:
```suggestion
            mock_openapi_to_yaml = mocker.patch("scanapi.openapi_to_yaml")
```

I checked out your code and it works without the `main`